app/models/manageiq/providers/ibm_power_vc/cloud_manager.rb

```diff
@@ -370,6 +370,14 @@ def self.params_for_create
                     :label     => _("PowerVC Server Password"),
                     :type      => "password",
                   },
+
+                  {
+                    :component => "password-field",
+                    :id        => "authentications.node.options",
+                    :name      => "authentications.node.options",
```

Comment:
I'm not sure that `Authentication` is the right place to store this since it isn't related to the credential at all...
Maybe `Endpoint#path` even though that is really intended for the path component of a URI

Reply:
maybe a new hash field called `Endpoint#options`?

Reply:
@chessbyte do you mean modifying Endpoint class itself for that?

Reply:
@iv1111 Yes, update the Endpoint schema with something like [this](https://github.com/ManageIQ/manageiq-schema/blob/master/db/migrate/20171219161635_create_transformation_mappings.rb#L8)

Reply:
https://github.com/ManageIQ/manageiq-schema/pull/595

Reply:
@agrare thanks for solving it in the meantime, i tested it, now storing in _endpoints.node.options_.